src/modules/dashboard/components/ColonyActions/ColonyActions.tsx

```diff
@@ -38,7 +41,7 @@ const MSG = defineMessages({
   },
   noActionsFound: {
     id: 'dashboard.ColonyActions.noActionsFound',
-    defaultMessage: `The colony did not create any actions yet.
+    defaultMessage: `There are no actions yet.
```

Comment:
Thought it might be good to change it to a generic phrase that goes for both colony actions and domain actions

Reply:
I mean we can even remove it.

Since after everything is wired up, all colonies will have at least one action (the first domain created), so you'll see this reflected in my #2348 

But yeah, leave the change in, it doesn hurt :)